databroker/src/broker.rs

```diff
@@ -417,12 +438,53 @@ impl Entry {
         }
     }
 
+    /// Returns true if min/max conditions is violated
+    /// value is supposed to be of the base type, like Int32 for an Int32Array
+    fn validate_value_min_max_violated(&self, value: &DataValue) -> bool {
```

Comment:
The name of this function in combination with a boolean return value makes it quite likely to be misunderstood in my opinion.

I think it would be safer to rename it and have it return a `Result<(), ValidationError>` or something.. That would make it hard to misunderstand.

Reply:
Refactored